papaparse.js

```diff
@@ -226,6 +227,10 @@
 			else
 				streamer = new StringStreamer(_config);
 		}
+		else if (_input.readable === true && typeof _input.read === 'function' && typeof _input.on === 'function')
```

Comment:
Is there any special reason to add this before the global.File If?

For me it sounds more logical to add new features at the end.